Sources/YPersistence/Manager/PersistenceManager+Sync.swift

```diff
@@ -63,7 +65,100 @@ extension PersistenceManager {
         )
         return try fetchRecords(
             predicate: predicate,
-            sortDescriptors: T.sort?.descriptors
+            sortDescriptors: T.sort?.descriptors,
+            context: context
         )
     }
+
+    /// Update records that have been marked for upload since the last sync.
+    ///
+    /// May be safely called from any thread
+    /// may only be safely accessed from the current thread.
+    /// If no context is provided, a worker context will be created.
+    /// - Parameters:
+    ///   - enity: the Core Data entity to be fetched
+    ///   - uids: array of uids to update
+    ///   - context: Optional managed context to use. Default is `nil`.
+    /// - Throws: any error executing the fetch or save request
+    public func markRecordsAsUploaded<T: SyncRecord>(
+        enity: T.Type,
```

Comment:
```suggestion
        entity: T.Type,
```
Typo. Also in the doc comments.